apiserver/crossmodel/crossmodel.go

```diff
@@ -239,7 +261,7 @@ func makeOfferParamsFromOffer(offer jujucrossmodel.ApplicationOffer) params.Appl
 // FindApplicationOffers gets details about remote applications that match given filter.
 func (api *API) FindApplicationOffers(filters params.OfferFilters) (params.FindApplicationOffersResults, error) {
 	var result params.FindApplicationOffersResults
-	offers, err := api.getApplicationOffersDetails(filters)
+	offers, err := api.getApplicationOffersDetails(filters, permission.ReadAccess)
```

Comment:
seems weird that Find and List have different perms. can you please leave a comment why. I assume it's because the details of offers, only returned by List, are sensitive.

(it also seems weird that List returns details, and Find doesn't...)